pycaching/geocaching.py

```diff
@@ -364,47 +380,58 @@ def search_quick(self, area, *, strict=False, zoom=None):
 
     # add some shortcuts ------------------------------------------------------
 
-    def search_rect(self, rect: Rectangle, *, per_query: int = 50, sort_by: str = 'datelastvisited', origin=None):
+    def search_rect(
+        self,
+        rect: Rectangle,
+        *,
+        per_query: int = 50,
+        sort_by: Union[str, SortOrder] = SortOrder.date_last_visited,
+        origin: Optional[Point] = None
+    ):
         """
         Return a generator of caches in given Rectange area.
 
         :param rect: Search area.
-        :param per_query: Number of caches requested in single query.
+        :param int per_query: Number of caches requested in single query.
         :param sort_by: Order cached by given criterion.
-        :param origin: Origin point for search by distance.
+        :param origin: Origin point for search by distance
```

Comment:
Is there a reason for removing the full stop?

Reply:
nope, fixed